tests/scripts/all.sh

```diff
@@ -2667,10 +2667,9 @@ config_psa_crypto_config_accel_ecc_no_bignum() {
 
     # RSA support is intentionally disabled on this test because RSA_C depends
     # on BIGNUM_C.
-    scripts/config.py -f include/psa/crypto_config.h unset PSA_WANT_KEY_TYPE_RSA_KEY_PAIR_BASIC
-    scripts/config.py -f include/psa/crypto_config.h unset PSA_WANT_KEY_TYPE_RSA_KEY_PAIR_IMPORT
-    scripts/config.py -f include/psa/crypto_config.h unset PSA_WANT_KEY_TYPE_RSA_KEY_PAIR_EXPORT
-    scripts/config.py -f include/psa/crypto_config.h unset PSA_WANT_KEY_TYPE_RSA_KEY_PAIR_GENERATE
+    for KT in $(sed -n 's/^#define \(PSA_WANT_KEY_TYPE_RSA_[0-9A-Z_a-z]*\).*/\1/p' <"$CRYPTO_CONFIG_H"); do
```

Comment:
Note: `grep 'for\>.*\<sed' tests/scripts/all.sh` already finds 4 occurrences before this PR, so this is arguably pre-existing.

Reply:
This looks like an argument to add a feature to `config.py`.

Reply:
I was looking at the source of `config.py` and I found that actually it might already support what I was planning to implement, i.e. disabling more than 1 symbol at once simply by using some regex. The option is named `unset-all`.
I will give it a try and see if it fits our needs here